ankihub/gui/deck_browser.py

```diff
@@ -0,0 +1,105 @@
+"""Modifies the Anki deck browser (aqt.deck_browser)."""
```

Comment:
⛏️ The name of the file in the anki repo is `deckbrowser.py`:
https://github.com/ankitects/anki/blob/0986af4f81e59668cce049697f5a739bf9a86a58/qt/aqt/deckbrowser.py#L1

We can use `deck_browser.py` as the name of this file in the add-on repo, but we should fix the docstring:
```suggestion
"""Modifies the Anki deck browser (aqt.deckbrowser)."""
```